manifests/compiler.pp

```diff
@@ -1,6 +1,10 @@
+# == Class homebrew::compiler
+# 
+# Installs Command Line Tools from user provided package if OS version > 10.7
+# 
 class homebrew::compiler {
 
-  if $::has_compiler == "true" {
+  if $::has_compiler == true {
```

Comment:
This might be a breaking change, I had to explicitly do this to fix an issue where the hiera variable was being passed as a string.